.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "\U0001F41E Bug report"
+about: Let us know if something isn't working as you expect it to.
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
```

Comment:
When I [preview](https://github.com/WordPress/wordcamp.org/blob/3e573def4a32f2b974eaf39475eaab6ba0d49e25/.github/ISSUE_TEMPLATE/bug-report.md) all of these section headers are missing the blank line that they have here.

`SHOULD` - It seems like we should use `###` instead of `**` so that they're actual headings.

This applies to the other files as well.